TrackingTools/TrajectoryFiltering/interface/CkfBaseTrajectoryFilter.h

```diff
@@ -51,6 +51,18 @@ class CkfBaseTrajectoryFilter : public TrajectoryFilter {
   virtual bool toBeContinued( TempTrajectory& traj) const {return TBC<TempTrajectory>(traj);}
 
   virtual  std::string name() const { return "CkfBaseTrajectoryFilter";}
+
+  inline edm::ParameterSetDescription getFilledConfigurationDescription() {
+    edm::ParameterSetDescription descLooper           = theLooperTrajectoryFilter->getFilledConfigurationDescription();
+    edm::ParameterSetDescription descLostHitsFraction = theLostHitsFractionTrajectoryFilter->getFilledConfigurationDescription();
+    edm::ParameterSetDescription descMinHits          = theMinHitsTrajectoryFilter->getFilledConfigurationDescription();
+
+    edm::ParameterSetDescription desc;
+    desc.add<edm::ParameterSetDescription>("looperTrajectoryFilter",          descLooper);
+    desc.add<edm::ParameterSetDescription>("lostHitsFractionTrajectoryFilter",descLostHitsFraction);
+    desc.add<edm::ParameterSetDescription>("minHitsTrajectoryFilter",         descMinHits);
+    return desc;
+  }
```

Comment:
ciao
thanks for the remark
such a function is not called yet, 
I added it as a book keeper for trying to dealing w/ the fillDescription (as you catch ;) )
yes, it is adding an extra PSet, as now
instead of simply adding the parameters list
(as said this is just a book keeper)